graphlearn/python/nn/tf/layers/ego_gcn_conv.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2021 Alibaba Group Holding Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""EgoGraph based GCN convolutional layer"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+try:
+  # https://www.tensorflow.org/guide/migrate
+  import tensorflow.compat.v1 as tf
+  tf.disable_v2_behavior()
+except ImportError:
+  import tensorflow as tf
+
+from graphlearn.python.nn.tf.layers.ego_layer import EgoConv, EgoLayer
+from graphlearn.python.nn.tf.layers.linear_layer import LinearLayer
+
+
+class EgoGCNConv(EgoConv):
```

Comment:
It seems like EgoRGCNConv, not EgoGCN. You can use EgoSAGEConv with aggr='gcn' as EgoGCNConv.

Reply:
Okay. will test that. Thanks!